traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -350,6 +375,27 @@ func (s *TOServer) Update() (error, error, int) {
 		}
 	}
 
+	current := TOServer{}
+	err := s.ReqInfo.Tx.QueryRowx(selectV20UpdatesQuery() + ` WHERE sv.id=` + strconv.Itoa(*s.ID)).StructScan(&current)
```

Comment:
Thinking about sql injections here, should we make this query like this instead?
```
... +" WHERE sv.id = $1", *s.ID).StructScan...
```
Obviously an integer ID can never lead to sql injection, but it's probably good practice to use parameterized queries whenever possible.

Reply:
done